pkg/controller/buildrun/generate_taskrun.go

```diff
@@ -89,14 +89,14 @@ func GenerateTaskSpec(build *buildv1alpha1.Build, buildRun *buildv1alpha1.BuildR
 	}
 
 	if build.Spec.BuilderImage != nil {
-		InputBuilderImage := v1beta1.ParamSpec {
-				Description: "Image containing the build tools/logic",
-				Name:        inputParamBuilderImage,
-				Default: &v1beta1.ArrayOrString{
-					Type:      v1beta1.ParamTypeString,
-					StringVal: build.Spec.BuilderImage.ImageURL,
-				},
-			}
+		InputBuilderImage := v1beta1.ParamSpec{
```

Comment:
Missed a `blank` here: `v1beta1.ParamSpec{`

Reply:
I cannot see where

Reply:
Just this one: `v1beta1.ParamSpec{` -> `v1beta1.ParamSpec {`